static/app/components/profiling/ProfilingOnboarding/util.ts

```diff
@@ -15,30 +15,33 @@ export type SupportedProfilingPlatform = (typeof supportedProfilingPlatforms)[nu
 export type SupportedProfilingPlatformSDK =
   (typeof supportedProfilingPlatformSDKs)[number];
 
-const platformToDocsPlatformSDK: Record<
-  SupportedProfilingPlatform,
-  SupportedProfilingPlatformSDK
-> = {
-  android: 'android',
-  'apple-ios': 'apple-ios',
-  node: 'node',
-  'node-express': 'node',
-  'node-koa': 'node',
-  'node-connect': 'node',
-  python: 'python',
-  'python-django': 'python',
-  'python-flask': 'python',
-  'python-sanic': 'python',
-  'python-bottle': 'python',
-  'python-pylons': 'python',
-  'python-pyramid': 'python',
-  'python-tornado': 'python',
-  rust: 'rust',
-};
+function getDocsPlatformSDKForPlatform(platform): PlatformKey | null {
+  if (platform === 'android') {
+    return 'android';
+  }
+
+  if (platform === 'apple-ios') {
+    return 'apple-ios';
+  }
+
+  if (platform.startsWith('node')) {
+    return 'node';
+  }
```

Comment:
@JonasBa This is going to enable the onboarding for all node platforms. Le me know if this works for you.

Reply:
Works for me - I'd rather get users to test it and tell us if it doesnt work so we can look at supporting it